core_tic_tac_toe/input_validator.py

```diff
@@ -0,0 +1,29 @@
+import string
+
+
+class InputValidator:
+
+    def is_move_valid(self, input_move, board):
+        if not self.__contains_invalid_character(input_move) and input_move.isdigit():
+            player_input = int(input_move) - 1
+            return board.valid_move(player_input)
+        else:
+            return False
+
+    def __contains_invalid_character(self, input_move):
+        return self.__is_alpha(input_move) or self.__is_white_space(input_move) or self.__is_special_character(input_move) or self.__contains_space(input_move) or self.__is_empty_space(input_move)
```

Comment:
Instead of trying to check for all possible classes of invalid character, could you check for only valid characters?